core/pta/remote_attestation/cbor.c

```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2024, Institute of Information Security (IISEC)
+ */
+
+#include "cbor.h"
+#include "sign.h"
+#include <kernel/pseudo_ta.h>
+
+UsefulBufC
+encode_evidence_to_cbor(const char *eat_profile, const int psa_client_id,
+                        const int psa_security_lifecycle,
+                        const uint8_t *psa_implementation_id,
+                        size_t psa_implementation_id_len,
+                        const char *measurement_type, const uint8_t *signer_id,
+                        size_t signer_id_len, const uint8_t *psa_instance_id,
+                        size_t psa_instance_id_len, const uint8_t *psa_nonce,
+                        size_t psa_nonce_len, const uint8_t *measurement_value,
+                        size_t mv_len, UsefulBuf cbor_evidence_buffer) {
```

Comment:
Coding style:
```c
UsefulBufC
encode_evidence_to_cbor(const char *eat_profile, const int psa_client_id,
                        (...)
                        size_t mv_len, UsefulBuf cbor_evidence_buffer)
{
```
and use tabulation for indentation.

Reply:
DONE